ansible_builder/main.py

```diff
@@ -120,11 +131,30 @@ def __init__(self, definition,
         self.base_image = base_image
         self.container_runtime = container_runtime
         self.tag = tag
+        self.get_additional_commands = get_additional_commands
```

Comment:
Did you mean to remove this?

The method `UserDefinition.get_additional_commands` is important, and is clear. However, `AnsibleBuilder. get_additional_commands` and `Containerfile. get_additional_commands` are dynamically assigned, and I don't see it doing anything, it looks like it would always be `None`, and would never be referenced.